ml/encoders.py

```diff
@@ -29,27 +29,9 @@ def _init_weights(layer, initializer_range=0.02):
 
 
 class SharedEmbeddingEncoder(TorchEncoder, config_name='shared_embedding_encoder'):
-    """
```

Comment:
Cound you please return previous semantics for the `SharedEmbeddingEncoder`? It is more flexible